src/main/java/sirius/web/data/ExcelExport.java

```diff
@@ -52,12 +52,15 @@
 import java.util.Date;
 import java.util.HashSet;
 import java.util.Set;
+import java.util.function.Consumer;
 
 /**
  * Generates an Excel file which can be sent as a response for a {@link sirius.web.http.WebContext}
  */
 public class ExcelExport {
 
+    public static final int MAX_NUM_ROWS = 1_000_000;
```

Comment:
javadoc

Reply:
shouldn't we also have one for XLS?

Reply:
this limit applies for xls and xlsx

Reply:
oh okay XLS has a different limit, i'll add one 👌